examples/common_test.go

```diff
@@ -20,6 +25,16 @@ var args = map[string][]string{
 	"clone":       []string{defaultURL, tempFolder()},
 	"progress":    []string{defaultURL, tempFolder()},
 	"open":        []string{filepath.Join(cloneRepository(defaultURL, tempFolder()), ".git")},
+	"push": []string{
+		func() string {
+			remote := createBareRepository(tempFolder())
+			local := filepath.Join(cloneRepository(defaultURL, tempFolder()), ".git")
+			setRemote(local, fmt.Sprintf("file://%s", remote))
+			return local
+		}(),
+		"origin",
```

Comment:
the default RemoteName is "origin" should to omit this arg to simplify the example?